src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.logo__link {
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  margin-right: 50px;
+}
+
+.header__ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.header__li {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+}
+
+.header__li:first-of-type {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+  color: black;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  content: '';
+  bottom: 0;
+  display: block;
+  width: 37px;
+  height: 4px;
+  background-color: #00acdc;
```

Comment:
Consider using variables for repeating colors

Reply:
Not fixed